src/common/histogram.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "metric_description.hpp"
+#include "registry.hpp"
+
+namespace shar::metrics {
+
+using HistogramFamily = prometheus::Family<prometheus::Histogram>;
+
+struct HistogramRemover {
```

Comment:
Move to `private` section. It is implementation detail (as well as typedef above).